it/helpers/ComponentSpecBase.scala

```diff
@@ -73,7 +73,7 @@ trait ComponentSpecBase extends TestSuite with CustomMatchers
   object FinancialTransactions {
     def get(uri: String): WSResponse = await(buildClient(uri).get())
     def getFinancialTransactions(idType: String, id: String, queryParameters: FinancialRequestQueryParameters): WSResponse = {
-      val queryParamStart = if(queryParameters.hasQueryParameters) "?" else ""
+      val queryParamStart = if(queryParameters.queryParams1166.nonEmpty) "?" else ""
```

Comment:
Could you change this function so that `queryParameters` is now of type `Seq[(String, String)]` so that the 1166 or 1811 values can be passed in by the tests that need the appropriate set